.github/workflows/e2e-test.yml

```diff
@@ -37,7 +37,7 @@ jobs:
       matrix:
         # Min WooCommerce | L-2 | L(atest) | beta
         woocommerce:   [ '5.8.1', '6.4.1', '6.6.0', 'beta' ]
```

Comment:
@zmaglica Can you update the WC version `6.6.0` to `latest`, so we don't have to specify the latest version every time? The setup script has support for this.